test/unittests/commands/validate_data/test_query_validation_result.py

```diff
@@ -47,6 +50,16 @@ def test_reformat_validation_graph(report_target_resource_wrong_type: tuple[Grap
     assert result.expected == "Representation"
 
 
+def test_separate_bns_of_results(report_target_resource_wrong_type, report_not_resource):
+    val_g1, _ = report_target_resource_wrong_type
+    val_g2, _ = report_not_resource
+    combined_g = val_g1 + val_g2
```

Comment:
Here, type hints would make sense, I guess. without, it's difficult to grasp what's happening.